### PR TITLE
Refactor build scripts

### PR DESCRIPTION
**Pull request by gavinhenderson**
The build scripts have been refactored to:
* Not be dependent on each other
* Use handlebar partials

### CODE REVIEW

File: scripts/mkpdf.js [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
-  const CATEGORIES_PDF_FILE_NAME = 'categories/categories.pdf';
+  const CATEGORIES_PDF_FILE_NAME = path.resolve(
+    __dirname,
+    '..',
+    'categories/categories.pdf',
+  );
   const TEMPLATE_PATH = path.resolve(__dirname, 'templates/pdf-template.html');
 
+  const DIRECTORY_NAME = 'categories';
+  if (!fs.existsSync(DIRECTORY_NAME)) {
```
Comment (SteveALee) [inline scripts/mkpdf.js]: Seems you forgot to require this dep above. Everyhting works find when I added that. I can;t push to your PR so can you fixed it?

**Comment by SteveALee** [comment]
@gavinhenderson just pinging in case my above comment did not cause a notification

**Comment by gavinhenderson** [comment]
@SteveALee Thanks! Thats it sorted now

**Comment by SteveALee** [comment]
I see you removed the building message on the CLI but as the commands themselves are descriptive when printed that's fine

**Comment by SteveALee** [comment]
Wowzers - it seem my pushes to the local branch for your PR DID get pushed back to GitHub. The docs say wouldn't work unless you turned on the option when you created the PR:) Plus I couldn;t see how to push back in VS code - but just a normal sync did it. Clever

**Comment by SteveALee** [comment]
Wowzers - it seem my pushes to the local branch for your PR DID get pushed back to GitHub. The docs say wouldn't work unless you turned on the option when you created the PR:) Plus I couldn;t see how to push back in VS code - but just a normal sync did it. Clever

**Comment by SteveALee** [comment]
Wowzers - it seem my pushes to the local branch for your PR DID get pushed back to GitHub. The docs say wouldn't work unless you turned on the option when you created the PR:) Plus I couldn;t see how to push back in VS code - but just a normal sync did it. Clever

**Comment by gavinhenderson** [comment]
@SteveALee Just pinging you as you might have left a tab open that keeps reposing a comment :+1:

**Comment by SteveALee** [comment]
Bah - Firefox seems to start up wit hthat tab in edit mode - so I saved to be safe. I assumed it would not post duplicate. Thanks for the heads up!

**Pull request merged**